bin/do_cf1d.py

```diff
@@ -139,6 +140,8 @@ def cf1d(p):
         cfs = map(cf1d,keys)
     else: cfs = map(cf1d,data.keys())
```

Comment:
@londumas you still need to replace map() by pool.map() otherwise it doesn't change anything ;^)

Reply:
just did and indeed now it multithread

Reply:
still getting the same results?